go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -118,13 +113,13 @@ func EnsureLBs(nbClient libovsdbclient.Client, service *corev1.Service, LBs []LB
 	for k, v := range addLBsToSwitch {
 		ops, err = libovsdbops.AddLoadBalancersToLogicalSwitchOps(nbClient, ops, getSwitch(k), v...)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to add load balancers to switches for service %s/%s: %w", service.Namespace, service.Name, err)
```

Comment:
these error messages need to have context around ops. The load balancers were not added to the switches...should be something like:

"failed to create ops for adding load balancers to switches for service..."